app/src/main/java/io/appium/uiautomator2/model/settings/MjpegFiltering.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.uiautomator2.model.settings;
+
+import io.appium.uiautomator2.server.ServerConfig;
+
+public class MjpegFiltering extends AbstractSetting<Boolean> {
+    private static final String SETTING_NAME = "mjpegFiltering";
```

Comment:
how about renaming this to `mjpegUseBilinearFiltering`?

Reply:
That seems like a better / more descriptive name

Reply:
Going to to drop "use" from it however, because it looks really weird for the getters / setters

Reply:
This will be resolved w/ the upcoming update(s)